backend/src/app.ts

```diff
@@ -24,10 +26,12 @@ mongoose.connect(mongoURI, {}, () => {
 });
 
 server.app.use(json());
+server.app.set('view engine', 'pug')
```

Comment:
What's this for?

Reply:
This is so it can render the pug file that is being used to template the Javascript string which is the html file.